log-service/src/main/kotlin/org/vivlaniv/nexohub/Application.kt

```diff
@@ -94,36 +99,39 @@ fun main() {
         log.info("Batch of $writtenRows log messages inserted")
     }
 
+    launch(Dispatchers.IO) {
+        while (true) {
+            delay(30_000)
```

Comment:
Also it is better to specify duration unit explicitly. [Non-blocking delay](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/delay.html.) also exists, so we can wrap it in a simple launch.